kernel/Makefile

```diff
@@ -88,21 +88,17 @@ $(BUILD_DIR)/%.o: %.cpp $(HEADERS)
 	@$(OBJ_DIRS_MAKE)
 	@printf "$(COLOR_COM)(CXX)$(COLOR_NONE)\t$(shell basename $@)\n"
 	@$(CXX) $(CPPFLAGS) $(CFLAGS) $(CXXFLAGS) -std=c++17 -MMD -c -o $@ $<
-# GAS assembly -> object
-$(BUILD_DIR)/%.o: %.s
-	@$(OBJ_DIRS_MAKE)
-	@printf "$(COLOR_COM)(AS)$(COLOR_NONE)\t$(shell basename $@)\n"
-	@$(AS) $(ASFLAGS) -o $@ $<
 # NASM assembly -> object
 $(BUILD_DIR)/%.o: %.S
 	@$(OBJ_DIRS_MAKE)
 	@printf "$(COLOR_COM)(NASM)$(COLOR_NONE)\t$(shell basename $@)\n"
 	@$(NASM) -f elf32 -o $@ $<
+# Kernel object alias
+$(KERNEL): $(PRODUCTS_DIR)/$(MODE)/$(KERNEL)
```

Comment:
I know we didn't have it before, but this really should be `PHONY`

Reply:
Just tested it again -- making this phony breaks things. Don't know why, just know it does.

Reply:
Super strange, I guess just leave it then